src/components/radiobutton/radiobutton.component.ts

```diff
@@ -33,8 +33,11 @@ export class RadioButtonComponent<T = any> implements ControlValueAccessor {
     /** Specify if this is a required input */
     @Input() required: boolean;
 
-    /** Specify the tabindex */
-    @Input() tabindex: number = 0;
+    /**
+     * Specify the tabindex
+     * @deprecated This input is deprecated and will be removed in the next major release.
+     */
```

Comment:
I noticed that you have reinstated `tabindex` in the template. Is it still appropriate to remove this input?

Reply:
Yeah, ideally the tabindex will be handled by the `_internalTabIndex` but if anyone wishes to use this input for now it can remain deprecated but it really isn't that useful, now the original issues have been resolved.